src/main/java/org/ccci/util/mail/MailMessage.java

```diff
@@ -38,22 +40,31 @@
 public class MailMessage 
 {
     private Session session;
-    private Message message;
+    private MimeMessage message;
 
     private Logger log = Logger.getLogger(MailMessage.class);
     
     private List<InternetAddress> toList = Lists.newArrayList();
 
     private boolean sent = false;
+    private Charset charset = null;
     
     /** whether this message is an application message, or a system message */
     private final boolean system;
     
     MailMessage(Session session, boolean system)
     {
         this.session = session;
+        this.system = system;
         message = new MimeMessage(session);
+    }
+
+    MailMessage(Session session, boolean system, Charset charset)

```

Comment:
Any value calling this here https://github.com/CruGlobal/util/blob/master/src/main/java/org/ccci/util/mail/ErrorEmailer.java#L18?